service/AppDynamicsService.go

```diff
@@ -31,12 +31,27 @@ func SendAppdynamicsAlert(message string) {
 		return
 	}
 
+	if dat["business"] !=nil {
```

Comment:
This is not the correct way to check if a map contains an element. 
`if val, ok := dict["foo"]; ok {
    //do something here
}
`